.gitlab-ci.yml

```diff
@@ -1044,17 +1044,14 @@ test_full_integration:
     variables:
       - $RUN_INTEGRATION_TESTS == "true"
   stage: test
-  image: docker:18-dind
+  image: docker:19-dind
```

Comment:
I've have ported this (together with the rest of the pipeline) at #430. Please have a look.